pom.xml

```diff
@@ -164,8 +164,14 @@
       </dependency>
       <dependency>
         <groupId>com.hotels</groupId>
-        <artifactId>hcommon-ssh</artifactId>
-        <version>${hcommon-ssh.version}</version>
+        <artifactId>hcommon-hive-metastore</artifactId>
+        <version>${hcommon-hive-metastore.version}</version>
+        <exclusions>
```

Comment:
Just curious, what's pulling in a wrong version of guava? Hive via a transitive dependency?

Reply:
yes this seems wrong though, in dependency management. Everything in hcommon-hive-metastore is provided. Is this really needed?

Reply:
If possible, remove this please.